node/src/Pos/Client/Txp/Util.hs

```diff
@@ -505,7 +505,7 @@ stabilizeTxFee linearPolicy utxo outputs = do
         Nothing -> throwError FailedToStabilize
         Just tx -> pure $ tx & \(S.Min (S.Arg _ txRaw)) -> txRaw
   where
-    firstStageAttempts = 2 * length utxo + 5
+    firstStageAttempts = 2 * (min 200 $ length utxo) + 5
```

Comment:
To avoid execution in `O ( |utxo| )`. Only possible drawback of doing less iterations is slightly higher fees. I don't expect it be crucial in parctice